examples/rest_api/mlflow_tracking.py

```diff
@@ -0,0 +1,71 @@
+"""
+This simple example shows how we should use MLFlow Rest API for creating new runs and log parameters/metrics.
+For more details see: https://www.mlflow.org/docs/latest/rest-api.html 
+"""
+
+
+import os
+import time
+import requests
+
+#change this base URL so that it points to the host where your MLFlow server is running
+BASE_URL = 'http://localhost:5000/api/2.0/preview/mlflow'
```

Comment:
Can you make the host and port (i.e. localhost:5000) be an optional command line argument? Then it would be easier to test this with a server running somewhere else.